README.md

```diff
@@ -7,12 +7,66 @@ This is a CDK Project for spinning up a [Valheim](https://store.steampowered.com
 Uses [valheim-server-docker](https://github.com/lloesche/valheim-server-docker) - thanks to lloesche for putting it together!
 
 ## Table of Contents
-- [Installation](#installationdeployment)
-- [Configuration](#configuration)
-- [Cost Information](#solution-cost-information)
-- [FAQ](#common-problemsfaq)
-  * [Find your server IP](#how-do-i-find-the-ip-of-my-server)
-
+- [valheim-ecs-fargate-cdk](#valheim-ecs-fargate-cdk)
+  - [Table of Contents](#table-of-contents)
+  - [Fresh Installation](#fresh-installation)
+  - [Installation/Deployment](#installationdeployment)
+  - [Configuration](#configuration)
+  - [Solution Cost Information](#solution-cost-information)
+  - [Common Problems/FAQ](#common-problemsfaq)
+    - [How do I find the IP of my server?](#how-do-i-find-the-ip-of-my-server)
+      - [Via the ecs-cli:](#via-the-ecs-cli)
+      - [Via the AWS Console:](#via-the-aws-console)
+  - [To-Do](#to-do)
+
+## Fresh Installation
+1. To follow [Installation/Deployment](#installationdeployment), you will first need to install the AWS CLI alongside the AWS CDK. The below will show you how to do so. 
+2. First we need to install Node.js which also install NPM. You can download it [here](https://nodejs.org/en/)
+3. Once Node is installed you can then install the AWS CLI from [here](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+   1. Make sure you install the AWS CLI here "C:\Program Files\Amazon\AWSCLIV2"
+   2. For other Operating Systems, it should be the same instructions below, but you'll need to research the documentation corresponding to the OS Install.
+   3. Next let's open up our AWS Console, where we will create a user in IAM that can create and interact with the CDK and CLI
+   4. Go to this [link](https://console.aws.amazon.com/iam/home#/home)
+   5. Click "Users" on the left hand side
```

Comment:
Can we link to an existing set of instructions for this? Ala: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds

Reply:
Or can we move these to a separate README page with a link?

Reply:
Added. re-pulling